integrated_channels/canvas/client.py

```diff
@@ -747,7 +853,18 @@ def _handle_canvas_assignment_submission(self, grade, course_id, assignment_id,
                 'posted_grade': grade
             }
         }
+        start_time = time.time()
         submission_response = self.session.put(submission_url, json=submission_data)
+        duration_seconds = time.time() - start_time
+        stringify_and_store_api_record(
+            enterprise_customer=self.enterprise_configuration.enterprise_customer,
+            enterprise_customer_configuration_id=self.enterprise_configuration.id,
+            endpoint=submission_url,
+            data=json.dumps(submission_data),
```

Comment:
`stringify_and_store_api_record` is responsible for stringifying the `data`

Reply:
yes, submission_data can be sent directly.